bsky-cli/src/runner.rs

```diff
@@ -387,13 +386,20 @@ impl Runner {
                             .upload_blob(buf)
                             .await
                             .expect("upload blob");
-                        images.push(
-                            api::app::bsky::embed::images::ImageData {
-                                alt: image
+                        let alt= match args.alt_text.get(idx) {
```

Comment:
I suggest adding `.filter(|s| !s.is_empty()` or something like that, since an empty string will be assumed to be unspecified and the default filename will be used.